integration-tests/tests/example-page.cy.ts

```diff
@@ -0,0 +1,62 @@
+import { checkErrors } from '../support';
+
+const PLUGIN_TEMPLATE_NAME = 'console-plugin-template';
+const PLUGIN_TEMPLATE_PULL_SPEC = Cypress.env('PLUGIN_TEMPLATE_PULL_SPEC');
+export const isLocalDevEnvironment =
+  Cypress.config('baseUrl').includes('localhost');
+
+const installHelmChart = () => {
+  cy.exec(
+    `cd ../../console-plugin-template && /tmp/helm upgrade -i ${PLUGIN_TEMPLATE_NAME} charts/openshift-console-plugin -n ${PLUGIN_TEMPLATE_NAME} --create-namespace --set plugin.image=${PLUGIN_TEMPLATE_PULL_SPEC}`,
```

Comment:
@jhadvig The `export` cmd seems not to work for me in the test script, so I used the path -  `/tmp/helm`. The `upgrade -i` cmd upgraded and deployed helm chart successfully in local.

![Screenshot 2023-01-19 at 4 54 25 PM](https://user-images.githubusercontent.com/15249132/213577750-1da24ff4-4776-4934-a222-0dc99c1e294e.png)
cal.